index.html

```diff
@@ -129,7 +129,335 @@ <h2 class="section-title">Schedule</h2>
       <!-- Begin "FAQ" -->
       <section id="faq" class="row">
         <h2 class="section-title">FAQ</h2>
+        <div class="accordion accordion-flush" id="accordionFlushExample">
```

Comment:
I would suggest changing the `id` away from the default example. Make sure to reflect this change in the `data-bs-parent=#accordionFlushExample` property for each FAQ card as well.

```suggestion
        <div class="accordion accordion-flush" id="faq-accordion">
```